src/app/components/data-entry-grid/index.html

```diff
@@ -113,6 +113,11 @@ <h2>ag-Grid styles</h2>
         To add the SKY UX styles for ag-Grid to your SPA, include <sky-code>"@skyux/ag-grid/css/sky-ag-grid.css"</sky-code> in the <sky-code>styles</sky-code> section of your <sky-code>skyuxconfig.json</sky-code>. For editable grids, include the class <sky-code>sky-ag-grid-editable</sky-code> on the <sky-code>ag-grid-angular</sky-code> element.
       </p>
 
+      <h2>ag-Grid wrapper component</h2>
+      <p>
+        The <sky-code>sky-ag-grid-wrapper</sky-code> component should wrap all uses of <sky-code>ag-grid-angular</sky-code>. The wrapper provides WCAG adherent keyboard navigation and sticky column headers for grids using auto height. It does not constrain standard ag-Grid functionality.
```

Comment:
I'd move the explanation of what the wrapper does to the first sentence. Also, maybe compliant instead of adherent?

The <sky-code>sky-ag-grid-wrapper</sky-code> component provides WCAG compliant keyboard navigation and sticky column headers for grids using auto height. Use the <sky-code>sky-ag-grid-wrapper</sky-code> component to wrap all uses of <sky-code>ag-grid-angular</sky-code> without constraining standard ag-Grid functionality.